packages/common/src/ui/components/navigation/AppNavLink/AppNavLink.stories.tsx

```diff
@@ -1,17 +1,16 @@
 import React, { FC, useEffect } from 'react';
 import { ComponentMeta, Story } from '@storybook/react';
 import { Route } from 'react-router-dom';
-
-import { NavLink, NavLinkProps } from './NavLink';
+import { Box } from '@mui/material';
+import { AppNavLink, AppNavLinkProps } from './AppNavLink';
```

Comment:
Renamed `NavLink` to `AppNavLink` because I'm exporting * from `react-router-dom` which has a `NavLink` .. seemed better?